client/package.go

```diff
@@ -1,3 +1,3 @@
 package client
 
-//go:generate go run github.com/oapi-codegen/oapi-codegen/v2/cmd/oapi-codegen --config=config.yaml ../../spec/openapi-3.0.json
+//go:generate go run github.com/oapi-codegen/oapi-codegen/v2/cmd/oapi-codegen --config=config.yaml ../spec/openapi-3.0.json
```

Comment:
The old version didn't work for me. Not sure if it's a difference in systems thing

```bash
andy@andy-desktop:~/Documents/projects/artifactsmmo-cli$ make generate-client
go generate ./...
error loading swagger spec in ../../spec/openapi-3.0.json
: open ../../spec/openapi-3.0.json: no such file or directoryexit status 1
client/package.go:3: running "go": exit status 1
make: *** [Makefile:7: generate-client] Error 1
```